File: hapiloImService/src/main/java/com/zongke/hapiloimservice/db/DBHelper.java
package com.zongke.hapiloimservice.db;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

/**
 * 数据库帮助类
 * Created by llj on 2017/10/9.
 */

public class DBHelper extends SQLiteOpenHelper {

    // 数据库名称
    private static final String DATABASE_NAME = "AccountDB";

    // 数据库版本
    private static final int DATABASE_VERSION = 1;


    public DBHelper(Context context) {
        super(context, DATABASE_NAME, null, DATABASE_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase) {
        sqLiteDatabase.execSQL(AccountColumns.CREATE_ACCOUNT_TABLE);
    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {

    }
}

File: app/src/main/java/com/zongke/hapilolauncher/db/DBHelper.kt
package com.zongke.hapilolauncher.db

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper


/**
 * Created by ${xinGen} on 2017/7/7.
 * 数据库
 */

class DBHelper(context: Context) : SQLiteOpenHelper(context, Data.SQLITE_NAME, null, Data.SQLITE_VERSON) {


    override fun onCreate(sqLiteDatabase: SQLiteDatabase) {
        sqLiteDatabase.execSQL(Data.CREATE_TABLE_COLLECTION_APP)
    }

    override fun onUpgrade(sqLiteDatabase: SQLiteDatabase, i: Int, i1: Int) {

    }
}
